CMakeCommon.cmake

```diff
@@ -57,49 +57,49 @@ function (SetCompilerOptions target acVersion)
     
 endfunction ()
 
-function (LinkGSLibrariesToProject acVersion devKitDir addOnName)
+function (LinkGSLibrariesToProject target acVersion devKitDir)
```

Comment:
What was the reason behind changing the order of parameters? Now I'm struggling with this since it has to be modified everywhere we use the tools. Can we change this back?

Reply:
Is this still a problem you face? I guess now other usages are adapted to this order..